### DISCUSSION TITLE
Just starting with vcpkg trying with zlib

### DISCUSSION THREAD

**Discussion by perdrix52** [microsoft/vcpkg#42544] [General]
I just set this up to work with my MSBuild projects, and it built and installed zlib just find, but AFIACT it created no pdb files even though it build the debug version of the library ...

Is this normal?

**Comment by Osyotr** [comment]
PDB files for dynamic triplets are generated, but they are not copied into your output directory.
If you use static triplets, PDB files are not generated. The debug information is embedded in .lib files instead.

**Comment by perdrix52** [comment]
Thanks a lot for the explanation.  David
